maintainers/maintainer-list.nix

```diff
@@ -1891,6 +1891,16 @@
     githubId = 3392199;
     name = "Calum MacRae";
   };
+  cmars = {
```

Comment:
Please put this into a separate commit with the message ``maintainers: add NAME`` and sort it alphabetically.

Reply:
Done.